crates/sui-analytics-indexer/src/handlers/transaction_handler.rs

```diff
@@ -179,22 +185,37 @@ impl TransactionHandler {
 
             gas_price: txn_data.gas_price(),
 
-            raw_transaction: Base64::encode(bcs::to_bytes(&txn_data).unwrap()),
+            bcs_length: bcs::to_bytes(&txn_data).unwrap().len() as u64,
```

Comment:
We probably also want to have an "effects_bcs_length" and "events_bcs_length" and "signatures_bcs_length" in addition to the transaction bcs length. That would allow us to have a full picture of the size of a transaction.